q2cli/commands.py

```diff
@@ -23,13 +23,29 @@ class RootCommand(click.MultiCommand):
         ('dev', q2cli.dev.dev)
     ])
 
+    def __init__(self, *args, **kwargs):
+        # 'help' is `None`, drop it to supply our own.
+        if 'help' in kwargs:
+            del kwargs['help']
+        super().__init__(help=ROOT_COMMAND_HELP, *args, **kwargs)
```

Comment:
So any extra arguments passed to 
```python
@click.command(cls=q2cli.commands.RootCommand, invoke_without_command=True,
               no_args_is_help=True)
```
are sent to the constructor, which means you should be able to just add `help=` right there.